.github/workflows/benchmark.yml

```diff
@@ -17,7 +17,8 @@ jobs:
       - name: Set up go
         uses: actions/setup-go@v5
         with:
-          go-version: "1.22"
+          check-latest: true
```

Comment:
If we always pull the latest go version, I think we could have some similar that we had with the github actions, where our [Dockerfile](https://github.com/buildpacks/pack/pull/2273/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557) could stay behind right?